### ISSUE TITLE
crash on command-D

### ISSUE THREAD

**Issue by cmyr** [xi-editor/xi-mac#376]
This is on master-master; if I key <kbd>cmd</kbd><kbd>D</kbd> on any word in a new window I'll get a crash. This is new in the past week or so, so maybe related to #317?

**Comment by scholtzan** [comment]
Oh no. I'll take a look.

**Comment by scholtzan** [comment]
Mh, currently, I cannot reproduce this. So you are just opening a new window and do cmd+D?

**Comment by cmyr** [comment]
yea... it's possible I'm running a weird combination of branches, but shouldn't be any changes in xi-mac? Let me try again, i'm on a different computer now.

**Comment by scholtzan** [comment]
Oh, now I can reproduce it. Was on the wrong branch. And yes, looks like a marker bar issue
